### ISSUE TITLE
New Feature: Option to "rescan from block #" for a specific wallet

### ISSUE THREAD

**Issue by kdmukai** [cryptoadvance/specter-desktop#27]
Some xpubs are likely to have existing historical transactions (especially USB hardware wallets) that the bitcoin node's wallet would be unaware of without doing a deeper historical `rescanblockchain <height>`.

Could either be on the wallet creation screen or maybe an option on the wallet settings page.

Maybe best to hide more advanced features like this on the wallet settings page to reduce UI clutter.

**Comment by kdmukai** [comment]
Depends on #24.

**Comment by kdmukai** [comment]
Superseded by #35 with more background info.
